core/cas-server-core-util/src/test/java/org/apereo/cas/util/RegexUtilsTests.java

```diff
@@ -0,0 +1,28 @@
+package org.apereo.cas.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests for {@link RegexUtils}
+ *
+ * @author David Rodriguez
+ * @since 5.0.0
+ */
+public class RegexUtilsTests {
+
+    @Test
+    public void verifyNotValidRegex() throws Exception {
```

Comment:
2. Does not matter. Anything should do.
3. Service id of the service you just created.